components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIProvider.java

```diff
@@ -91,7 +91,7 @@ Comment getComment(ApiTypeWrapper apiTypeWrapper, String commentId, Integer repl
      * @return boolean
      * @throws APIManagementException if failed to delete comment for identifier
      */
-    boolean deleteComment(ApiTypeWrapper apiTypeWrapper, String commentId) throws APIManagementException;
+    boolean deleteComment(String apiTypeWrapper, String commentId) throws APIManagementException;
```

Comment:
variable name should be uuid instead of apiTypeWrapper

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/10614/commits/b561990adfc350c2aec76fc8e72c88c6ad896f15